packages/android_alarm_manager/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
```

Comment:
Did you mean to add this file?

Reply:
no, i will fix it now and push again , Thank you @dnfield

Reply:
it's good, I took care of all the comments